tests/containers/search/filters/location/LocationSearchContainer-test.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * LocationSearchContainer-test.jsx
+ * Created by Emily Gullo 09/22/2017
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { LocationSearchContainer } from
+    'containers/search/filters/location/LocationSearchContainer';
+
+import { mockApi } from './mockLocations';
+
+const setupShallow = (props) => shallow(<LocationSearchContainer {...props} />);
+
+const parsedApi = {
+    location: {
+        matched_ids: [1, 2, 3],
+        parent: "INDIANA",
+        place: "PAWNEE",
+        place_type: "CITY"
+    }
+};
+
+const toggleValue = {
+    target: {
+        value: 'foreign'
+    }
+};
+
+describe('LocationSearchContainer', () => {
+    describe('selectLocation', () => {
+        it('should update the location name in redux', () => {
```

Comment:
What about when the location is not valid or no location is provided? Tests should cover as many cases as possible so we can identify when the code breaks on these cases (and on which cases it broke).